web-components/navdots/navdots.svelte

```diff
@@ -23,21 +23,20 @@
 </script>
 
 <nav class="leo-navdots" aria-label={label}>
-  <ol class="dot-container" part="dot-container" bind:this={container}>
```

Comment:
Note: I had a talk with Pete a while back and we're not going to expose `part`s until we need them :smile: In addition, these were failing the new version of Svelte Check